core/src/main/java/com/tweet/core/model/Tweet.java

```diff
@@ -1,11 +1,18 @@
 package com.tweet.core.model;
 
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDate;
 
 @Entity
 public class Tweet {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long tweetId;
     private String username;
     private String content;
-    private Long date;
+    private LocalDate createdDate;
```

Comment:
worth adding getters and setters (in both entities), in most cases classes should be immutable but entity is an excteption